networkx/algorithms/tests/test_covering.py

```diff
@@ -1,3 +1,5 @@
+from pytest import raises
```

Comment:
Just a nit, but the general best-practice is to simply import `pytest` and access the functionality from the `pytest` namespace, for example `pytest.raises`. One reason for doing so is that `raises` looks an awful lot like the Python builtin `raise`, which can easily confuse a reader who is skimming the code.

Reply:
Sounds good, I'll do that in the future